beanstalk_worker/worker_start_scan.py

```diff
@@ -507,6 +508,60 @@ def process_output(self, logs):
 
         return data
 
+
+class ContainerCapabilities(Scanner):
+    def __init__(self):
+        self.scanner_name = "container-capabilities-scanner"
+        self.full_scanner_name = \
+            "registry.centos.org/pipeline-images/container-capabilities-scanner"
+        self.scan_types = ["check-capabilities"]
+
+    def run(self, image_under_test):
+        # initializing a blank list that will contain results from all the
+        # scan types of this scanner
+        logs = []
+        super(ContainerCapabilities, self).__init__(
+            image_under_test=image_under_test,
+            scanner_name=self.scanner_name,
+            full_scanner_name=self.full_scanner_name,
+            to_process_output=False
```

Comment:
We could but `image_under_test` value is provided only at the time of `run` function. Not at the time of creating the object.